scripts/lisdf_plan_to_reset.py

```diff
@@ -1,7 +1,8 @@
 """Create a command that resets the robot from its current state to the initial
 state of another LISDF plan.
 
-Prepend that "reset" command with the other LISDF plan to create a final plan.
+Prepend that "reset" command with the other LISDF plan to create a final
```

Comment:
revert.

Reply:
I can't this is unfortunately something the autoformatter just changed lol...